docker/docker-compose.ci.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+
```

Comment:
there are no volumes for the database and test container, this is because we are using both containers for testing and no state is persisted